src/table.ts

```diff
@@ -1507,13 +1508,26 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     const onBatchResponse = (
       err: ServiceError | PartialFailureError | null
     ) => {
-      // TODO: enable retries when the entire RPC fails
       if (err) {
-        // The error happened before a request was even made, don't retry.
+        // Retry RPC level errors
+        if (!(err instanceof PartialFailureError)) {
+          const serviceError = err as ServiceError;
+          if (
+            numRequestsMade <= maxRetries &&
+            IDEMPOTENT_RETRYABLE_STATUS_CODES.has(serviceError.code)
+          ) {
+            console.log('RETRYING ' + err.code);
+            makeNextBatchRequest();
```

Comment:
Im a bit concerned that there is no backoff...it can result in a death spiral for the service

Reply:
Should we add a fixed delay for now?